test/unit/lib/DiscoveryStreamFeed.test.js

```diff
@@ -569,6 +569,36 @@ describe("DiscoveryStreamFeed", () => { // eslint-disable-line max-statements
     });
   });
 
+  describe("#clearSpocs", () => {
+    it("should not fail with no endpoint", async () => {
+      sandbox.stub(feed.store, "getState").returns({
+        Prefs: {
+          values: {"discoverystream.endpointSpocsClear": null},
+        },
+      });
+      sandbox.stub(feed, "fetchFromEndpoint").resolves(null);
+
+      await feed.clearSpocs();
+
+      assert.notCalled(feed.fetchFromEndpoint);
+    });
+    it("should call DELETE with endpoint", async () => {
+      sandbox.stub(feed.store, "getState").returns({
+        Prefs: {
+          values: {"discoverystream.endpointSpocsClear": "https://spocs/user"},
+        },
+      });
+      sandbox.stub(feed, "fetchFromEndpoint").resolves(null);
```

Comment:
Not sure if you were intentionally stubbing `fetchFromEndpoint` instead of `fetch` to bypass the security check

Reply:
Yeah, there is a lot happening in fetch that I didn't want to test.

I think either or would have worked, but by the time this patch fires `fetchFromEndpoint` I wasn't noticing anything unique about it after that point.

I also didn't change anything inside `fetchFromEnpoint` so I figured keep the test constrained to this spot.